### PR TITLE
Add Article_Type__c and Topics__c to Resource__kav

### PR DESCRIPTION
**Pull request by prescod**
1. Add Article_Type__c and Topics__c to Resource__kav

2. Move the data from the HTML to WLMA via those two properties

3. Add a logging statement to help debug an indexing issue in Sfdoc -> WLMA.

4. Add a test for the new properties.

### CODE REVIEW

File: sfdoc/publish/html.py [hunk 1]
```diff
@@ -24,15 +24,17 @@ def __init__(self, htmlpath, rootpath):
...
             ('url_name', 'UrlName', False),
             ('summary', 'description', True),
             ('is_visible_in_csp', 'is-visible-in-csp', False),
             ('is_visible_in_pkb', 'is-visible-in-pkb', False),
             ('is_visible_in_prm', 'is-visible-in-prm', False),
             ('author', settings.ARTICLE_AUTHOR, False),
-            ('docset_id', 'ProductMapUUID', True)  # Should be required but will break a lot of tests,
+            ('docset_id', 'ProductMapUUID', True),  # Should be required but will break a lot of tests,
+            ('Topics__c', "HubTopics", True),
+            ('Article_Type__c', "ArticleType", True),
```
Comment (davisagli) [inline sfdoc/publish/html.py]: Is making these new attributes required going to create any backwards-compatibility problems when publishing existing docs?

File: sfdoc/publish/html.py [hunk 2]
```diff
@@ -83,6 +85,8 @@ def create_article_data(self):
             'IsVisibleInCsp': self.is_visible_in_csp,
             'IsVisibleInPkb': self.is_visible_in_pkb,
             'IsVisibleInPrm': self.is_visible_in_prm,
+            "Article_Type__c": self.Article_Type__c,
+            "Topics__c": self.Topics__c,
```
Comment (davisagli) [inline sfdoc/publish/html.py]: Do these fields already exist in production WLMA, or do we need to coordinate the timing of merging this?

File: sfdoc/publish/html.py (hunk 1 above)
Comment (prescod) [reply sfdoc/publish/html.py]: Sorry: the comment is confusing. I'll fix that. True means optional.

File: sfdoc/publish/html.py (hunk 2 above)
Comment (prescod) [reply sfdoc/publish/html.py]: Yeah, I'm talking to Dave Lea about that in Quip#sfdoc. He says:

> Article_Type__c is available, Topics__c won't be available in production until tomorrow around 8pm EST ish

So yeah, I guess we'll deploy SFdoc on Wednesday morning.

**Pull request merged**